client/src/elm/Measurement/View.elm

```diff
@@ -1425,8 +1425,8 @@ viewVitalsForm language currentDate config form =
                 (Maybe.map toFloat form.heartRate)
                 (config.setIntInputMsg heartRateUpdateFunc)
                 "heart-rate"
-                Translate.BpmUnitLabel
-            , Pages.Utils.viewPreviousMeasurement language config.heartRatePreviousValue Translate.BpmUnitLabel
+                Translate.HeartBpmUnitLabel
```

Comment:
Lets rename `BpmUnitLabel` to `BreathsPerMinuteUnitLabel` and use it for respiratory rate.
For heart, lest use `BeatsPerMinuteUnitLabel`.